src/plugins/dashboard/public/application/lib/sync_dashboard_container_input.ts

```diff
@@ -112,7 +107,6 @@ export const applyContainerChangesToState = ({
   if (!compareFilters(input.filters, filterManager.getFilters(), COMPARE_ALL_OPTIONS)) {
     // Add filters modifies the object passed to it, hence the clone deep.
     filterManager.addFilters(_.cloneDeep(input.filters));
-    applyFilters(latestState.query, input.filters);
```

Comment:
Why is this no longer necessary?

Reply:
The `applyFilters` function was just something that I built to simplify applying the filters to the saved dashboard. It was never something that was required when syncing filters or anything